README.md

```diff
@@ -6,39 +6,111 @@ Please edit this README.md file with your answers to the worksheet questions.
+|    0    |    0    |    0    |    1       |    1         |    0     |    0                        |
+|    1    |    1    |    1    |    0       |    1         |    1     |    1                        |
+|    1    |    1    |    0    |    0       |    1         |    1     |    1                        |
+|    1    |    0    |    0    |    1       |    1         |    1     |    1                        |
+|    0    |    1    |    1    |    0       |    0         |    1     |    0                        |
+|    0    |    0    |    1    |    1       |    1         |    1     |    1                        |
+|    1    |    0    |    1    |    1       |    1         |    1     |    1                        |
+|    0    |    1    |    0    |    0       |    0         |    0     |    0                        |
 ### d
-
+|    A    |    B    |    C    |    D    |  NOT B   |   NOT C |  NOT A   |   NOT(B OR NOT C) |   NOT A AND D |   A AND NOT(B OR NOT C) AND (NOT A AND D) |
+|---------|---------|---------|---------|----------|---------|----------|-------------------|---------------|-------------------------------------------|
+|    0    |    0    |    0    |    0    |    1     |    1    |    1     |    1              |    0          |    0                                      |
+|    1    |    1    |    1    |    1    |    0     |    0    |    0     |    0              |    0          |    0                                      |
+|    1    |    1    |    1    |    0    |    0     |    0    |    0     |    0              |    0          |    0                                      |
+|    1    |    1    |    0    |    0    |    0     |    1    |    0     |    1              |    0          |    0                                      |
+|    1    |    0    |    0    |    0    |    1     |    1    |    0     |    1              |    0          |    0                                      |
+|    0    |    1    |    1    |    1    |    0     |    0    |    1     |    0              |    1          |    0                                      |
+|    0    |    0    |    1    |    1    |    1     |    0    |    1     |    1              |    1          |    0                                      |
+|    0    |    0    |    0    |    1    |    1     |    1    |    1     |    1              |    1          |    0                                      |
+|    0    |    1    |    1    |    0    |    0     |    0    |    1     |    0              |    0          |    0                                      |
+|    1    |    0    |    0    |    1    |    1     |    1    |    0     |    1              |    1          |    1                                      |
+|    0    |    1    |    0    |    0    |    0     |    1    |    1     |    1              |    0          |    0                                      |
+|    0    |    0    |    1    |    0    |    1     |    0    |    1     |    1              |    0          |    0                                      |
```

Comment:
You seem to have missed a few lines in this table (there are 4 variables, so there should be 2^4 = 16 rows). Counting up in binary from 0000 to 1111 is the best way to make sure you cover all the possibilities.